### PR TITLE
Redirect deprecated routes

### PR DESCRIPTION
**Pull request by rowanmanning**
This redirects file, module, and documentation routes in the same way as bundle routes are redirected.

### CODE REVIEW

File: lib/express/v1-deprecated.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use strict';
```
Comment (AlbertoElias) [inline lib/express/v1-deprecated.js]: what about putting this in the /deprecated folder?

File: lib/express/v1-deprecated.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+const hostnames = require('../utils/hostnames');
+const redirectWithBody = require('../express/redirect-with-body');
+const URL = require('url');
+
+module.exports = function(req, res) {
+	const redirectUrl = '/v2' + URL.parse(req.url).path.replace(/^\/v1/, '');
+	const redirectBody = 'This endpoint has been deprecated. You are being redirected to ' + redirectUrl + '\nSee https://' + hostnames.preferred + '/v2/#api-reference for more information.\n';
```
Comment (AlbertoElias) [inline lib/express/v1-deprecated.js]: more than deprecatated, this is now removing it

File: lib/express/v1-deprecated.js (hunk 1 above)
Comment (rowanmanning) [inline lib/express/v1-deprecated.js]: Good idea, I've moved it

File: lib/express/v1-deprecated.js (hunk 2 above)
Comment (rowanmanning) [inline lib/express/v1-deprecated.js]: Good point. I've tweaked the messaging

File: test/integration/v1-files.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const request = require('supertest');
+
+describe('GET /v1/files', function() {
```
Comment (AlbertoElias) [inline test/integration/v1-files.js]: I'd personally find it easier if this was in the files.js but in a different describe, but I don't have a strong opinion about it

**Comment by AlbertoElias** [comment]
Looks good! I don't think this should break in any way, files and modules haven't changed

File: test/integration/v1-files.js (hunk 3 above)
Comment (rowanmanning) [inline test/integration/v1-files.js]: Ah I'm trying to match the filenames in here to the URLs

Comment (AlbertoElias) [inline test/integration/v1-files.js]: Yup, and that's fine as well, just wanted to mention my personal preference :P

**Pull request merged**